### PR TITLE
Updated_khandler&Tests 

### PR DESCRIPTION
**Pull request by lukefarn1996**
Updated KHandler and updated the tests to now contain a test to check for comma delimited .k files with examples.

### CODE REVIEW

**Comment by lukefarn1996** [comment]
Hi @mtezzele, Managed to fix some of the tests however i keep getting "The command "conda info -a" failed and exited with 1 during ." from Travis in the build process.

**Comment by lukefarn1996** [comment]
UpdateTest

File: tests/test_khandler.py [hunk 1]
```diff
@@ -117,3 +117,28 @@ def test_k_write_comparison_2(self):
         k_handler.write(mesh_points, outfilename)
         self.assertTrue(filecmp.cmp(outfilename, outfilename_expected))
         self.addCleanup(os.remove, outfilename)
+
+    def test_comma_seperated_parse(self):
+        k_handler = uh.KHandler()
+        mesh_points = k_handler.parse('tests/test_datasets/test_square_comma.k')
+        np.testing.assert_almost_equal(mesh_points[0][0], -0.0500000007)
```
Comment (mtezzele) [inline tests/test_khandler.py:124]: please merge these 3 lines in only one assert

**Comment by mtezzele** [comment]
> Hi @mtezzele, Managed to fix some of the tests however i keep getting "The command "conda info -a" failed and exited with 1 during ." from Travis in the build process.

We are investigating this problem. I'll keep you updated.

**Comment by mtezzele** [comment]
I should have solved the conda problem with #189. @lukefarn1996 with a fetch+rebase this PR should pass.

**Comment by mtezzele** [comment]
Perfect, now the last thing @lukefarn1996 : can you do a fetch+rebase and/or a squash of your commits, please?

**Comment by e-dub** [comment]
Hi @mtezzele and @lukefarn1996: I am trying to get my LS-Dyna tutorials working again and the stopper are the changes discussed here to the khandler.  The previous code works still while this does not.  Is this an operating system issue? I am in Linux, you in Windows?

**Comment by mtezzele** [comment]
@e-dub please open an issue with a snippet of code replicating the error you get. Thanks.

**Comment by e-dub** [comment]
@mtezzele: See [tutorial-6-k.py](https://github.com/e-dub/PyGeM/blob/master/tutorials/tutorial-6-k.py)
It is a script version of the tutorial.  

The new version of the khandler gives the following error:
```
  File ~/opt/PyGeM/tutorials/tutorial-6-k.py", line 14, in <module>
    new_mesh_points = ffd(mesh_points)

  File "/usr/local/lib/python3.8/dist-packages/pygem-2.0.0-py3.8.egg/pygem/ffd.py", line 560, in __call__
    src_reference_frame_pts = self.psi(src_pts - self.box_origin)

ValueError: operands could not be broadcast together with shapes (0,) (3,) 
```

With the old version of the khandler it works
[khandler.zip](https://github.com/mathLab/PyGeM/files/5327206/khandler.zip)

**Comment by mtezzele** [comment]
@e-dub Open an issue please. You can link this PR from the issue. We should not continue here since this PR is closed.

**Pull request merged**